site/book/04-using-functions/01-declarative-function-execution.md

```diff
@@ -178,5 +178,43 @@ pipeline:
         tier: mysql
 ```
 
+### `selectors`
```

Comment:
This doesn't belong under "Specifying `functionConfig`", it should be unleveled as "Specifying `selectors`". General guideline, make sure any edits match what comes before and after.